assets/js/persistence.js

```diff
@@ -79,8 +80,8 @@ function persistChoices(mode, updatePermalink) {
     }
 
     if(window.history.replaceState && parent.location.hash) {
-        var hash = parent.location.hash,
-            urlParams = [],
+        hash = parent.location.hash;
```

Comment:
Are you saying something like I could use `var hash` inside any function as required? and thus not make it global? Eg: inside any function where it is required, I would define a new variable as follows:

`var hash = parent.location.hash;`